fuel-crypto/src/secp256/backend/r1/p256.rs

```diff
@@ -99,7 +99,7 @@ mod tests {
         let signing_key = SigningKey::random(&mut rng);
         let verifying_key = signing_key.verifying_key();
 
-        let message = Message::new([rng.gen(); 100]);
+        let message = Message::new([rng.r#gen(); 100]);
```

Comment:
Very sad that now we need to add it everywhere. Cursio to see patch for the trait itself to change the name